9.2-2.3/alpine/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM postgres:9.2-alpine
+MAINTAINER Régis Belson <me@regisbelson.fr>
+
+ENV POSTGIS_VERSION 2.3.1
+ENV POSTGIS_SHA256 0f1e0fe08f55b6db791d8600bb343e4f77590725acbbec99094a30b8b7439cd8
+
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/main >> /etc/apk/repositories
```

Comment:
@tclarke @ncopa It looks like you guys are the maintainers of the `gdal` and `proj4` Alpine packages respectively, and I know that you @ncopa have been involved in the `postgresql:alpine` Docker image.

Do either of you have a sense of whether `geos`, `gdal`, and `proj4` will make it out of `edge` into a numbered release any time soon? Also, I'm curious to get your thoughts @ncopa on the drawbacks of running an `apk upgrade`. I'm hesitant to have that in the `Dockerfile` since it seems like it could result in an inconsistent build artifact (not to mention bloat from upgrading packages found in the base `alpine:3.5` or `postgres:*-alpine` image).